Pages/Pages.css

```diff
@@ -0,0 +1,92 @@
+.Pages{
+    display: grid;
+    width:100%;
+    grid-template-columns:1fr 1fr 1fr ;
+    grid-auto-columns: 1fr ;
```

Comment:
Can you tell please tell me why you are using ```grid-auto-column```  when you already have a specified template for columns. I did not understand this.